### PR TITLE
add GetStaticIPs function and test case

### PR DESCRIPTION
**Pull request by benben**
## Affected Components
* [x] New Features

## Style
* [x] Go code is formatted with `go fmt`

This adds a new function `GetStaticIPs` to expose our current static IP ranges in the SDK. Will later need this for the terraform provider.

### CODE REVIEW

File: types.go [hunk 1]
```diff
@@ -912,6 +914,12 @@ type Runtime struct {
 	Description      string `json:"description"`
 }
 
+type StaticIP struct {
+	Value  string
```
Comment (danielpaulus) [inline types.go:918]: how about making this a net.IP ?

File: types.go [hunk 2]
```diff
@@ -912,6 +914,12 @@ type Runtime struct {
...
+	Family string
```
Comment (danielpaulus) [inline types.go:919]: Family is not needed if you have a net.IP, you can tell which family is it then using a golang function

Comment (danielpaulus) [reply types.go:919]: https://pkg.go.dev/net/netip#Addr.Is4

File: types.go (hunk 1 above)
Comment (benben) [reply types.go:918]: not sure. this would be mostly just nice but not needed since it will end up as strings anyways in the tf provider.

Comment (danielpaulus) [reply types.go:918]: true, but there are people using the go-sdk directly and I like it when SDKs come with good devex.

File: types.go [hunk 3]
```diff
@@ -912,6 +914,12 @@ type Runtime struct {
 	Description      string `json:"description"`
 }
 
+type StaticIP struct {
```
Comment (danielpaulus) [inline types.go:917]: would be also nice to add a comment explaining what this type is for

File: types.go (hunk 1 above)
Comment (benben) [reply types.go]: done

File: types.go (hunk 2 above)
Comment (benben) [reply types.go]: done

**Review by danielpaulus — APPROVED** [review]

**Pull request merged**